pyfolio/plotting.py

```diff
@@ -896,7 +896,8 @@ def plot_rolling_beta(returns, factor_returns, legend_loc='best',
     return ax
 
 
-def plot_rolling_volatility(returns, rolling_window=APPROX_BDAYS_PER_MONTH * 6,
+def plot_rolling_volatility(returns, factor_returns=pd.DataFrame(),
```

Comment:
Ah, need to fix the doc-string above. And here would be great if you were to fix that.